stablehlo/tests/interpret_is_finite.mlir

```diff
@@ -0,0 +1,43 @@
+// RUN: stablehlo-translate --interpret -split-input-file %s
```

Comment:
Hello @sdasgup3 , this might be a very silly question but could you tell me what ODS means ? And how that ODS functions are written i.e. what guidelines are being used ?

Reply:
I should do a better job in using acronym for the first time. ODS (operation definition specification) is the specification of operations and types using tablegen syntax. More details can be found at https://mlir.llvm.org/docs/DefiningDialects/Operations/

Reply:
Hi @karthikrangasai 
May be this is still unresolved. Can we keep just one test in this file, same as the one introduced in the spec.md?

Reply:
@sdasgup3 , I have updated the test file to have only that one single test from the `spec.md` file.

Reply:
Thanks!